:  failed

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247605?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247605?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,081 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 24s processwrapper-sandbox ... (8 actions, 7 running)
[2,098 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 25s processwrapper-sandbox ... (8 actions, 7 running)
[2,108 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 26s processwrapper-sandbox ... (7 actions, 6 running)
[2,124 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 27s processwrapper-sandbox ... (8 actions, 7 running)
[2,135 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 28s processwrapper-sandbox ... (8 actions, 7 running)
[2,149 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 29s processwrapper-sandbox ... (8 actions, 7 running)
[2,155 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 30s processwrapper-sandbox ... (8 actions, 7 running)
[2,178 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 31s processwrapper-sandbox ... (8 actions, 7 running)
[2,187 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 32s processwrapper-sandbox ... (8 actions, 7 running)
[2,199 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 34s processwrapper-sandbox ... (7 actions, 6 running)
[2,213 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 35s processwrapper-sandbox ... (8 actions, 7 running)
[2,232 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 36s processwrapper-sandbox ... (8 actions, 7 running)
[2,251 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 37s processwrapper-sandbox ... (8 actions, 7 running)
[2,262 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 38s processwrapper-sandbox ... (8 actions, 7 running)
[2,271 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 39s processwrapper-sandbox ... (8 actions, 7 running)
[2,280 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 40s processwrapper-sandbox ... (6 actions running)
[2,286 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 41s processwrapper-sandbox ... (5 actions running)
[2,302 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 42s processwrapper-sandbox ... (5 actions, 4 running)
[2,317 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 43s processwrapper-sandbox ... (8 actions, 7 running)
[2,326 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 44s processwrapper-sandbox ... (5 actions, 3 running)
[2,335 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 46s processwrapper-sandbox ... (2 actions running)
[2,341 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 47s processwrapper-sandbox ... (2 actions running)
[2,346 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 48s processwrapper-sandbox ... (2 actions, 1 running)
[2,348 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 49s processwrapper-sandbox ... (2 actions, 1 running)
[2,349 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 51s processwrapper-sandbox ... (2 actions running)
[2,350 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 52s processwrapper-sandbox ... (2 actions, 1 running)
[2,351 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 54s processwrapper-sandbox ... (2 actions running)
[2,352 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 62s processwrapper-sandbox ... (2 actions, 1 running)
[2,353 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 64s processwrapper-sandbox
[2,353 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 73s processwrapper-sandbox
[2,353 / 2,368] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 103s processwrapper-sandbox
[2,354 / 2,368] [Prepa] Executing genrule //pkg/sql/lexbase:gen-tokens
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg700429250 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2254/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/ses
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-36409

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247609?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247609?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,718 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 125s processwrapper-sandbox ... (8 actions running)
[2,733 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 126s processwrapper-sandbox ... (8 actions, 7 running)
[2,750 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 127s processwrapper-sandbox ... (8 actions, 7 running)
[2,757 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 128s processwrapper-sandbox ... (8 actions, 7 running)
[2,769 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 129s processwrapper-sandbox ... (8 actions, 7 running)
[2,779 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 130s processwrapper-sandbox ... (8 actions, 7 running)
[2,794 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 131s processwrapper-sandbox ... (8 actions, 7 running)
[2,805 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 133s processwrapper-sandbox ... (8 actions, 7 running)
[2,809 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc; 131s processwrapper-sandbox ... (8 actions, 7 running)
[2,820 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc; 132s processwrapper-sandbox ... (8 actions, 7 running)
[2,826 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc; 133s processwrapper-sandbox ... (8 actions, 7 running)
[2,841 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc; 134s processwrapper-sandbox ... (8 actions, 7 running)
[2,854 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc; 135s processwrapper-sandbox ... (8 actions, 7 running)
[2,862 / 3,578] Executing genrule //pkg/sql/parser:sql-goyacc; 136s processwrapper-sandbox ... (8 actions, 7 running)
[2,885 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/encoder.a; 0s processwrapper-sandbox ... (8 actions running)
[2,899 / 3,578] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 1s processwrapper-sandbox ... (7 actions running)
[2,914 / 3,578] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 2s processwrapper-sandbox ... (8 actions, 7 running)
[2,923 / 3,578] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 3s processwrapper-sandbox ... (8 actions, 7 running)
[2,936 / 3,578] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 4s processwrapper-sandbox ... (8 actions, 7 running)
[2,948 / 3,578] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 5s processwrapper-sandbox ... (8 actions, 7 running)
[2,967 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color_indent/vm_color_indent.a; 5s processwrapper-sandbox ... (7 actions, 6 running)
[2,982 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color_indent/vm_color_indent.a; 6s processwrapper-sandbox ... (8 actions, 7 running)
[3,001 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color_indent/vm_color_indent.a; 7s processwrapper-sandbox ... (6 actions, 5 running)
[3,023 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color_indent/vm_color_indent.a; 9s processwrapper-sandbox ... (2 actions, 1 running)
[3,028 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color/vm_color.a; 0s processwrapper-sandbox ... (8 actions running)
[3,049 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color/vm_color.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[3,067 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color/vm_color.a; 2s processwrapper-sandbox ... (7 actions, 6 running)
[3,076 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color/vm_color.a; 3s processwrapper-sandbox ... (8 actions, 7 running)
[3,085 / 3,578] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm_color/vm_color.a; 5s processwrapper-sandbox ... (7 actions, 6 running)
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg3114496004 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_migration.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_revival_token.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/local_only_session_data.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/sequence_cache.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/sequence_cache_node.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2897/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/session_data.go: exit status 2
Target //pkg/util/log/logcrash:logcrash_test failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /go/src
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247752?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247752?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,255 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 20s processwrapper-sandbox ... (8 actions, 7 running)
[2,267 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 21s processwrapper-sandbox ... (8 actions, 7 running)
[2,282 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 22s processwrapper-sandbox ... (8 actions, 7 running)
[2,302 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 23s processwrapper-sandbox ... (8 actions, 7 running)
[2,320 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 24s processwrapper-sandbox ... (8 actions, 7 running)
[2,352 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 25s processwrapper-sandbox ... (8 actions, 7 running)
[2,362 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 26s processwrapper-sandbox ... (8 actions running)
[2,381 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 27s processwrapper-sandbox ... (8 actions running)
[2,406 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 28s processwrapper-sandbox ... (8 actions, 7 running)
[2,425 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 29s processwrapper-sandbox ... (8 actions, 7 running)
[2,437 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 30s processwrapper-sandbox ... (8 actions, 7 running)
[2,446 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 31s processwrapper-sandbox ... (8 actions, 7 running)
[2,452 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 33s processwrapper-sandbox ... (8 actions, 7 running)
[2,462 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 34s processwrapper-sandbox ... (8 actions, 7 running)
[2,477 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 35s processwrapper-sandbox ... (8 actions, 7 running)
[2,492 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 36s processwrapper-sandbox ... (8 actions, 7 running)
[2,502 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 37s processwrapper-sandbox ... (8 actions running)
[2,523 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 38s processwrapper-sandbox ... (8 actions, 7 running)
[2,544 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 39s processwrapper-sandbox ... (8 actions, 7 running)
[2,565 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 40s processwrapper-sandbox ... (8 actions, 7 running)
[2,583 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 41s processwrapper-sandbox ... (8 actions, 7 running)
[2,598 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 42s processwrapper-sandbox ... (8 actions, 7 running)
[2,615 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 43s processwrapper-sandbox ... (8 actions, 7 running)
[2,630 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 44s processwrapper-sandbox ... (8 actions, 7 running)
[2,643 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 45s processwrapper-sandbox ... (8 actions, 7 running)
[2,654 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 47s processwrapper-sandbox ... (8 actions running)
[2,662 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 48s processwrapper-sandbox ... (8 actions, 7 running)
[2,681 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 49s processwrapper-sandbox ... (8 actions, 7 running)
[2,688 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 50s processwrapper-sandbox ... (8 actions, 7 running)
[2,699 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 51s processwrapper-sandbox ... (6 actions, 5 running)
[2,707 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 53s processwrapper-sandbox ... (4 actions, 3 running)
[2,707 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 54s processwrapper-sandbox ... (4 actions running)
[2,708 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 56s processwrapper-sandbox ... (4 actions, 3 running)
[2,715 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 57s processwrapper-sandbox ... (8 actions, 7 running)
[2,723 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 58s processwrapper-sandbox ... (4 actions running)
[2,724 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 59s processwrapper-sandbox ... (3 actions running)
[2,725 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 67s processwrapper-sandbox ... (3 actions, 2 running)
[2,735 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 68s processwrapper-sandbox ... (4 actions, 3 running)
[2,738 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 69s processwrapper-sandbox ... (3 actions, 2 running)
[2,739 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 70s processwrapper-sandbox ... (2 actions running)
[2,739 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 80s processwrapper-sandbox ... (2 actions running)
[2,739 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 110s processwrapper-sandbox ... (2 actions running)
[2,740 / 2,844] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 121s processwrapper-sandbox
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefine
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247758?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247758?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,200 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 35s processwrapper-sandbox ... (8 actions running)
[2,218 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 36s processwrapper-sandbox ... (8 actions, 7 running)
[2,226 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 37s processwrapper-sandbox ... (8 actions, 7 running)
[2,236 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 38s processwrapper-sandbox ... (8 actions, 7 running)
[2,245 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 39s processwrapper-sandbox ... (8 actions, 7 running)
[2,251 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 40s processwrapper-sandbox ... (8 actions, 7 running)
[2,259 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 41s processwrapper-sandbox ... (8 actions, 7 running)
[2,266 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 42s processwrapper-sandbox ... (8 actions, 7 running)
[2,286 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 43s processwrapper-sandbox ... (8 actions, 7 running)
[2,310 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 44s processwrapper-sandbox ... (8 actions, 7 running)
[2,320 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 45s processwrapper-sandbox ... (8 actions, 7 running)
[2,335 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 47s processwrapper-sandbox ... (8 actions, 7 running)
[2,354 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 48s processwrapper-sandbox ... (8 actions, 7 running)
[2,380 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 49s processwrapper-sandbox ... (8 actions, 7 running)
[2,400 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 50s processwrapper-sandbox ... (8 actions, 7 running)
[2,421 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 51s processwrapper-sandbox ... (8 actions, 7 running)
[2,435 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 52s processwrapper-sandbox ... (8 actions, 7 running)
[2,452 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 53s processwrapper-sandbox ... (8 actions running)
[2,465 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 54s processwrapper-sandbox ... (8 actions running)
[2,480 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 55s processwrapper-sandbox ... (7 actions, 6 running)
[2,494 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 57s processwrapper-sandbox ... (4 actions, 3 running)
[2,497 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 58s processwrapper-sandbox ... (4 actions running)
[2,504 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 59s processwrapper-sandbox ... (4 actions running)
[2,513 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 60s processwrapper-sandbox ... (3 actions, 2 running)
[2,514 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 62s processwrapper-sandbox ... (3 actions running)
[2,516 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 64s processwrapper-sandbox ... (2 actions, 1 running)
[2,523 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 66s processwrapper-sandbox ... (2 actions running)
[2,524 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 74s processwrapper-sandbox ... (2 actions, 1 running)
[2,528 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 76s processwrapper-sandbox
[2,528 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 88s processwrapper-sandbox
[2,528 / 2,558] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 118s processwrapper-sandbox
[2,529 / 2,558] [Prepa] Executing genrule //pkg/sql/lexbase:gen-tokens
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg2605982950 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_migration.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_revival_token.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2416/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/local_only_session_data.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247869?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14247869?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,890 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 109s processwrapper-sandbox ... (8 actions, 7 running)
[2,902 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 110s processwrapper-sandbox ... (8 actions, 7 running)
[2,915 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 111s processwrapper-sandbox ... (8 actions, 7 running)
[2,926 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 112s processwrapper-sandbox ... (8 actions, 7 running)
[2,930 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 114s processwrapper-sandbox ... (8 actions, 7 running)
[2,933 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 115s processwrapper-sandbox ... (7 actions, 6 running)
[2,937 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 116s processwrapper-sandbox ... (8 actions running)
[2,952 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 117s processwrapper-sandbox ... (8 actions, 7 running)
[2,966 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 118s processwrapper-sandbox ... (8 actions, 7 running)
[2,981 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 119s processwrapper-sandbox ... (8 actions, 7 running)
[2,997 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 120s processwrapper-sandbox ... (8 actions, 7 running)
[3,016 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 121s processwrapper-sandbox ... (8 actions, 7 running)
[3,030 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 122s processwrapper-sandbox ... (8 actions, 7 running)
[3,045 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 123s processwrapper-sandbox ... (8 actions, 7 running)
[3,056 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 124s processwrapper-sandbox ... (8 actions, 7 running)
[3,071 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 125s processwrapper-sandbox ... (8 actions, 7 running)
[3,081 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 126s processwrapper-sandbox ... (8 actions, 7 running)
[3,093 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 127s processwrapper-sandbox ... (8 actions, 7 running)
[3,098 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 129s processwrapper-sandbox ... (7 actions, 6 running)
[3,106 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 130s processwrapper-sandbox ... (8 actions, 7 running)
[3,110 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 131s processwrapper-sandbox ... (8 actions, 7 running)
[3,118 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 132s processwrapper-sandbox ... (8 actions running)
[3,127 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 133s processwrapper-sandbox ... (6 actions, 5 running)
[3,134 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 134s processwrapper-sandbox ... (8 actions, 7 running)
[3,144 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 135s processwrapper-sandbox ... (8 actions, 7 running)
[3,158 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 136s processwrapper-sandbox ... (5 actions, 4 running)
[3,161 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 138s processwrapper-sandbox ... (3 actions running)
[3,162 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 140s processwrapper-sandbox ... (4 actions, 2 running)
[3,170 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 141s processwrapper-sandbox ... (8 actions, 7 running)
[3,185 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 142s processwrapper-sandbox ... (8 actions, 7 running)
[3,192 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 143s processwrapper-sandbox ... (3 actions running)
[3,195 / 3,584] Executing genrule //pkg/sql/parser:sql-goyacc; 143s processwrapper-sandbox ... (5 actions running)
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2985/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg1929232529 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2985/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2985/execroot/com_github_cockroachdb_c
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248058?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248058?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,785 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 120s processwrapper-sandbox ... (8 actions, 7 running)
[2,792 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 121s processwrapper-sandbox ... (8 actions, 7 running)
[2,806 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 122s processwrapper-sandbox ... (8 actions, 7 running)
[2,809 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 123s processwrapper-sandbox ... (8 actions, 7 running)
[2,815 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 125s processwrapper-sandbox ... (8 actions, 7 running)
[2,817 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 126s processwrapper-sandbox ... (8 actions, 7 running)
[2,819 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 127s processwrapper-sandbox ... (8 actions running)
[2,827 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 128s processwrapper-sandbox ... (8 actions, 7 running)
[2,840 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 129s processwrapper-sandbox ... (6 actions, 5 running)
[2,842 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 130s processwrapper-sandbox ... (8 actions, 7 running)
[2,846 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 131s processwrapper-sandbox ... (8 actions, 7 running)
[2,851 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 133s processwrapper-sandbox ... (8 actions running)
[2,854 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 134s processwrapper-sandbox ... (8 actions running)
[2,858 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 135s processwrapper-sandbox ... (8 actions, 7 running)
[2,860 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 136s processwrapper-sandbox ... (8 actions, 7 running)
[2,867 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 138s processwrapper-sandbox ... (8 actions, 7 running)
[2,880 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 139s processwrapper-sandbox ... (8 actions, 7 running)
[2,897 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 140s processwrapper-sandbox ... (8 actions, 7 running)
[2,916 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 141s processwrapper-sandbox ... (6 actions, 5 running)
[2,928 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 142s processwrapper-sandbox ... (8 actions, 7 running)
[2,946 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 143s processwrapper-sandbox ... (8 actions running)
[2,959 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 144s processwrapper-sandbox ... (8 actions, 7 running)
[2,971 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 145s processwrapper-sandbox ... (8 actions, 7 running)
[2,985 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 146s processwrapper-sandbox ... (8 actions, 7 running)
[2,998 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 147s processwrapper-sandbox ... (8 actions, 7 running)
[3,016 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 148s processwrapper-sandbox ... (8 actions, 7 running)
[3,034 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 149s processwrapper-sandbox ... (8 actions, 7 running)
[3,050 / 3,581] Executing genrule //pkg/sql/parser:sql-goyacc; 150s processwrapper-sandbox ... (8 actions, 7 running)
[3,066 / 3,581] GoCompilePkg external/com_github_cockroachdb_pebble/sstable/sstable.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[3,087 / 3,581] GoCompilePkg external/com_github_cockroachdb_pebble/sstable/sstable.a; 2s processwrapper-sandbox ... (8 actions, 7 running)
[3,102 / 3,581] GoCompilePkg pkg/geo/geos/geos.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[3,108 / 3,581] GoCompilePkg pkg/geo/geos/geos.a; 2s processwrapper-sandbox ... (8 actions, 7 running)
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg2183627332 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_migration.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_revival_token.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/local_only_session_data.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2919/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/sequence_cache.go /home/roach/.cache/bazel/_bazel
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248219?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248219?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
INFO: Build options --//build/toolchains:crdb_test_flag, --@io_bazel_rules_go//go/config:race, --define, and 2 more have changed, discarding analysis cache.
Analyzing: target //pkg/util/tracing/collector:collector_test (0 packages loaded, 0 targets configured)
Analyzing: target //pkg/util/tracing/collector:collector_test (358 packages loaded, 13220 targets configured)
Analyzing: target //pkg/util/tracing/collector:collector_test (487 packages loaded, 14525 targets configured)
Analyzing: target //pkg/util/tracing/collector:collector_test (667 packages loaded, 19266 targets configured)
Analyzing: target //pkg/util/tracing/collector:collector_test (740 packages loaded, 22744 targets configured)
Analyzing: target //pkg/util/tracing/collector:collector_test (750 packages loaded, 25647 targets configured)
INFO: Analyzed target //pkg/util/tracing/collector:collector_test (1050 packages loaded, 29782 targets configured).
 checking cached actions
INFO: Found 1 test target...
[0 / 5] [Prepa] BazelWorkspaceStatusAction stable-status.txt
[695 / 2,971] checking cached actions
[1,539 / 3,582] GoStdlib external/io_bazel_rules_go/stdlib_/pkg; 0s processwrapper-sandbox
[2,384 / 3,582] GoStdlib external/io_bazel_rules_go/stdlib_/pkg; 1s processwrapper-sandbox
[2,384 / 3,582] GoStdlib external/io_bazel_rules_go/stdlib_/pkg; 12s processwrapper-sandbox
[2,385 / 3,582] [Prepa] GoCompilePkg external/com_github_getsentry_sentry_go/internal/debug/debug.a
[2,519 / 3,582] GoCompilePkg external/com_github_gogo_protobuf/proto/proto.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[2,532 / 3,582] GoCompilePkg external/com_github_gogo_protobuf/proto/proto.a; 2s processwrapper-sandbox ... (8 actions, 7 running)
[2,573 / 3,582] GoCompilePkg external/com_github_gogo_protobuf/proto/proto.a; 3s processwrapper-sandbox ... (8 actions, 7 running)
[2,663 / 3,582] GoCompilePkg external/com_github_gogo_protobuf/proto/proto.a; 4s processwrapper-sandbox ... (8 actions, 7 running)
[2,789 / 3,582] GoCompilePkg external/org_golang_x_text/unicode/norm/norm.a; 0s processwrapper-sandbox ... (7 actions, 6 running)
[2,815 / 3,582] GoCompilePkg external/com_github_gogo_protobuf/types/types.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[2,830 / 3,582] GoCompilePkg external/com_github_gogo_protobuf/types/types.a; 2s processwrapper-sandbox ... (7 actions, 6 running)
[2,890 / 3,582] GoCompilePkg external/com_github_apache_arrow_go_v11/arrow/array/array.a; 1s processwrapper-sandbox ... (5 actions running)
[2,895 / 3,582] GoCompilePkg external/com_github_apache_arrow_go_v11/arrow/array/array.a; 2s processwrapper-sandbox ... (8 actions running)
[2,984 / 3,582] GoCompilePkg external/com_github_apache_arrow_go_v11/arrow/array/array.a; 3s processwrapper-sandbox ... (3 actions running)
[3,025 / 3,582] GoCompilePkg external/com_github_apache_arrow_go_v11/arrow/array/array.a; 4s processwrapper-sandbox ... (4 actions running)
[3,032 / 3,582] GoCompilePkg external/com_github_apache_arrow_go_v11/parquet/internal/encoding/encoding.a; 0s processwrapper-sandbox ... (2 actions running)
[3,105 / 3,582] GoCompilePkg external/com_github_cockroachdb_pebble/pebble.a; 0s processwrapper-sandbox ... (8 actions, 7 running)
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg3040432846 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_migration.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_revival_token.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/local_only_session_data.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/sequence_cache.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/sequence_cache_node.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/352/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/session_data.go: exit status 2
Target //pkg/util/tracing/collector:collector_test failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/collector/BUILD.bazel:21:8 GoCompilePkg pkg/util/tracing/collector/collector_test~testmain.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/buil
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248217?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248217?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,965 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 151s processwrapper-sandbox ... (8 actions, 7 running)
[2,979 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 152s processwrapper-sandbox ... (8 actions, 7 running)
[2,990 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 153s processwrapper-sandbox ... (8 actions, 7 running)
[2,996 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 154s processwrapper-sandbox ... (8 actions, 7 running)
[3,004 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 155s processwrapper-sandbox ... (8 actions, 7 running)
[3,009 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 156s processwrapper-sandbox ... (8 actions, 7 running)
[3,018 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 157s processwrapper-sandbox ... (8 actions, 7 running)
[3,027 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 158s processwrapper-sandbox ... (8 actions running)
[3,033 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 159s processwrapper-sandbox ... (8 actions running)
[3,041 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 160s processwrapper-sandbox ... (8 actions, 7 running)
[3,052 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 161s processwrapper-sandbox ... (8 actions, 7 running)
[3,059 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 162s processwrapper-sandbox ... (8 actions, 7 running)
[3,066 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 163s processwrapper-sandbox ... (8 actions, 7 running)
[3,084 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 164s processwrapper-sandbox ... (8 actions, 7 running)
[3,102 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 165s processwrapper-sandbox ... (8 actions, 7 running)
[3,122 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 166s processwrapper-sandbox ... (8 actions, 7 running)
[3,133 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 168s processwrapper-sandbox ... (8 actions, 7 running)
[3,144 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 169s processwrapper-sandbox ... (8 actions, 7 running)
[3,159 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 170s processwrapper-sandbox ... (8 actions, 7 running)
[3,179 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 171s processwrapper-sandbox ... (8 actions, 7 running)
[3,193 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 172s processwrapper-sandbox ... (8 actions, 7 running)
[3,207 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 173s processwrapper-sandbox ... (7 actions, 6 running)
[3,216 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 174s processwrapper-sandbox ... (8 actions, 7 running)
[3,224 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 175s processwrapper-sandbox ... (8 actions, 7 running)
[3,229 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 176s processwrapper-sandbox ... (8 actions, 7 running)
[3,241 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 177s processwrapper-sandbox ... (7 actions running)
[3,253 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 178s processwrapper-sandbox ... (8 actions running)
[3,263 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 179s processwrapper-sandbox ... (8 actions, 7 running)
[3,277 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 180s processwrapper-sandbox ... (7 actions, 6 running)
[3,286 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 181s processwrapper-sandbox ... (8 actions, 7 running)
[3,297 / 4,106] Executing genrule //pkg/sql/parser:sql-goyacc; 182s processwrapper-sandbox ... (8 actions, 7 running)
[3,316 / 4,106] GoCompilePkg external/com_github_gogo_protobuf/types/types.a [for tool]; 1s processwrapper-sandbox ... (8 actions, 7 running)
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2722/execroot/com_github_c
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248434?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248434?buildTab=artifacts#/) on master @ [9ca613fe1a8b2331b8ed1af18bc59f509093cdbf](https://github.com/cockroachdb/cockroach/commits/9ca613fe1a8b2331b8ed1af18bc59f509093cdbf):


```
[2,678 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 131s processwrapper-sandbox ... (8 actions, 7 running)
[2,687 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 132s processwrapper-sandbox ... (8 actions running)
[2,692 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 133s processwrapper-sandbox ... (8 actions running)
[2,696 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 134s processwrapper-sandbox ... (8 actions, 7 running)
[2,709 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 135s processwrapper-sandbox ... (8 actions, 7 running)
[2,719 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 136s processwrapper-sandbox ... (8 actions, 7 running)
[2,725 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc; 137s processwrapper-sandbox ... (8 actions, 7 running)
[2,733 / 3,587] Executing genrule //pkg/sql/parser:sql-goyacc; 138s processwrapper-sandbox ... (8 actions, 7 running)
[2,742 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 8s processwrapper-sandbox ... (8 actions, 7 running)
[2,750 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 9s processwrapper-sandbox ... (8 actions running)
[2,756 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 10s processwrapper-sandbox ... (8 actions running)
[2,774 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 11s processwrapper-sandbox ... (7 actions, 6 running)
[2,793 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 12s processwrapper-sandbox ... (8 actions, 7 running)
[2,813 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 13s processwrapper-sandbox ... (8 actions, 7 running)
[2,833 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 14s processwrapper-sandbox ... (8 actions, 7 running)
[2,856 / 3,587] GoCompilePkg external/com_github_goccy_go_json/internal/encoder/vm/vm.a; 15s processwrapper-sandbox ... (8 actions, 7 running)
[2,876 / 3,587] GoCompilePkg external/com_github_google_pprof/internal/symbolz/symbolz.a; 0s processwrapper-sandbox ... (7 actions, 6 running)
[2,889 / 3,587] GoCompilePkg external/com_github_prometheus_procfs/procfs.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[2,905 / 3,587] GoCompilePkg external/com_github_apache_arrow_go_arrow/array/array.a; 0s processwrapper-sandbox ... (8 actions, 7 running)
[2,919 / 3,587] GoCompilePkg external/com_github_apache_arrow_go_arrow/array/array.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[2,928 / 3,587] GoCompilePkg external/org_golang_x_net/http2/http2.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[2,944 / 3,587] GoCompilePkg external/org_golang_x_net/http2/http2.a; 2s processwrapper-sandbox ... (6 actions, 5 running)
[2,954 / 3,587] GoCompilePkg external/io_vitess_vitess/go/vt/sqlparser/sqlparser.a; 3s processwrapper-sandbox ... (8 actions, 7 running)
[2,964 / 3,587] GoCompilePkg external/io_vitess_vitess/go/vt/sqlparser/sqlparser.a; 4s processwrapper-sandbox ... (8 actions, 7 running)
[2,981 / 3,587] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 2s processwrapper-sandbox ... (8 actions, 7 running)
[3,001 / 3,587] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 3s processwrapper-sandbox ... (8 actions, 7 running)
[3,023 / 3,587] GoCompilePkg external/com_github_dave_dst/decorator/decorator.a; 4s processwrapper-sandbox ... (7 actions, 6 running)
[3,041 / 3,587] GoCompilePkg pkg/util/tracing/tracingpb/tracingpb.a; 0s processwrapper-sandbox ... (8 actions, 7 running)
[3,058 / 3,587] GoCompilePkg external/com_github_cockroachdb_pebble/vfs/vfs.a; 0s processwrapper-sandbox ... (8 actions, 7 running)
[3,074 / 3,587] GoCompilePkg external/com_github_apache_arrow_go_v11/parquet/schema/schema.a; 0s processwrapper-sandbox ... (8 actions, 7 running)
[3,089 / 3,587] GoCompilePkg external/com_github_apache_arrow_go_v11/arrow/array/array.a; 1s processwrapper-sandbox ... (8 actions running)
[3,103 / 3,587] GoCompilePkg external/com_github_apache_arrow_go_v11/arrow/array/array.a; 2s processwrapper-sandbox ... (8 actions running)
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2900/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg4171595242 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2900/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2900/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2900/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2900/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2900/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2900/execr
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248648?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248648?buildTab=artifacts#/) on master @ [7345fae7a2449e17f1cf93dff8ae4089d5376a8c](https://github.com/cockroachdb/cockroach/commits/7345fae7a2449e17f1cf93dff8ae4089d5376a8c):


```
[1,845 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 15s processwrapper-sandbox ... (8 actions, 7 running)
[1,863 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 16s processwrapper-sandbox ... (8 actions, 7 running)
[1,875 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 17s processwrapper-sandbox ... (8 actions, 7 running)
[1,889 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 18s processwrapper-sandbox ... (8 actions, 7 running)
[1,907 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 19s processwrapper-sandbox ... (8 actions, 7 running)
[1,914 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 20s processwrapper-sandbox ... (8 actions, 7 running)
[1,925 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 21s processwrapper-sandbox ... (8 actions, 7 running)
[1,939 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 22s processwrapper-sandbox ... (8 actions, 7 running)
[1,952 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 23s processwrapper-sandbox ... (8 actions, 7 running)
[1,971 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 24s processwrapper-sandbox ... (8 actions, 7 running)
[1,996 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 25s processwrapper-sandbox ... (8 actions, 7 running)
[2,010 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 26s processwrapper-sandbox ... (8 actions, 7 running)
[2,025 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 27s processwrapper-sandbox ... (6 actions, 5 running)
[2,033 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 28s processwrapper-sandbox ... (8 actions, 7 running)
[2,045 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 29s processwrapper-sandbox ... (8 actions, 7 running)
[2,062 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 30s processwrapper-sandbox ... (8 actions, 7 running)
[2,074 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 32s processwrapper-sandbox ... (8 actions, 7 running)
[2,087 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 33s processwrapper-sandbox ... (7 actions, 6 running)
[2,097 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 34s processwrapper-sandbox ... (8 actions, 7 running)
[2,110 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 35s processwrapper-sandbox ... (8 actions, 7 running)
[2,125 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 36s processwrapper-sandbox ... (8 actions, 7 running)
[2,148 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 37s processwrapper-sandbox ... (8 actions, 7 running)
[2,163 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 38s processwrapper-sandbox ... (7 actions, 6 running)
[2,174 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 39s processwrapper-sandbox ... (8 actions, 7 running)
[2,192 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 40s processwrapper-sandbox ... (7 actions running)
[2,205 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 41s processwrapper-sandbox ... (4 actions, 3 running)
[2,210 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 43s processwrapper-sandbox ... (4 actions, 3 running)
[2,215 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 44s processwrapper-sandbox ... (2 actions running)
[2,217 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 46s processwrapper-sandbox
[2,217 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 57s processwrapper-sandbox
[2,217 / 2,229] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 87s processwrapper-sandbox
[2,218 / 2,229] [Prepa] Executing genrule //pkg/sql/lexbase:gen-tokens
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg3397769532 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_migration.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_revival_token.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2194/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/local_only_session_data.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandb
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248654?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248654?buildTab=artifacts#/) on master @ [7345fae7a2449e17f1cf93dff8ae4089d5376a8c](https://github.com/cockroachdb/cockroach/commits/7345fae7a2449e17f1cf93dff8ae4089d5376a8c):


```
[2,999 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 49s processwrapper-sandbox ... (8 actions running)
[3,003 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 51s processwrapper-sandbox ... (8 actions, 7 running)
[3,005 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 52s processwrapper-sandbox ... (8 actions, 7 running)
[3,007 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 53s processwrapper-sandbox ... (8 actions running)
[3,008 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 54s processwrapper-sandbox ... (8 actions running)
[3,012 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 56s processwrapper-sandbox ... (8 actions, 7 running)
[3,013 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 57s processwrapper-sandbox ... (8 actions running)
[3,014 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 58s processwrapper-sandbox ... (8 actions, 7 running)
[3,020 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 59s processwrapper-sandbox ... (6 actions, 5 running)
[3,042 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 60s processwrapper-sandbox ... (8 actions, 7 running)
[3,052 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 62s processwrapper-sandbox ... (8 actions, 7 running)
[3,060 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 63s processwrapper-sandbox ... (8 actions, 7 running)
[3,076 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 64s processwrapper-sandbox ... (8 actions, 7 running)
[3,086 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 65s processwrapper-sandbox ... (8 actions, 7 running)
[3,098 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 66s processwrapper-sandbox ... (8 actions, 7 running)
[3,105 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 67s processwrapper-sandbox ... (8 actions running)
[3,112 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 68s processwrapper-sandbox ... (8 actions running)
[3,116 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 69s processwrapper-sandbox ... (7 actions running)
[3,125 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 70s processwrapper-sandbox ... (8 actions, 7 running)
[3,138 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 71s processwrapper-sandbox ... (8 actions, 7 running)
[3,148 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 72s processwrapper-sandbox ... (8 actions, 7 running)
[3,156 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 74s processwrapper-sandbox ... (6 actions, 5 running)
[3,158 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 75s processwrapper-sandbox ... (4 actions running)
[3,165 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 76s processwrapper-sandbox ... (8 actions, 7 running)
[3,177 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 77s processwrapper-sandbox ... (8 actions, 7 running)
[3,190 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 79s processwrapper-sandbox ... (3 actions running)
[3,191 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 87s processwrapper-sandbox ... (3 actions, 2 running)
[3,200 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 88s processwrapper-sandbox ... (8 actions, 7 running)
[3,207 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 89s processwrapper-sandbox ... (3 actions running)
[3,210 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 91s processwrapper-sandbox ... (2 actions running)
[3,210 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 100s processwrapper-sandbox ... (2 actions running)
[3,211 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 110s processwrapper-sandbox
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2221/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg1757845246 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2221/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2221/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2221/execroot/com_github_cockroachdb_cockroach/bazel-out/
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248662?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248662?buildTab=artifacts#/) on master @ [7345fae7a2449e17f1cf93dff8ae4089d5376a8c](https://github.com/cockroachdb/cockroach/commits/7345fae7a2449e17f1cf93dff8ae4089d5376a8c):


```
[2,465 / 2,828] GoCompilePkg external/com_github_gogo_protobuf/types/types.a [for tool]; 2s processwrapper-sandbox ... (8 actions, 7 running)
[2,488 / 2,828] Executing genrule //pkg/kv/kvclient/kvcoord:gen-txnstate-stringer; 0s processwrapper-sandbox ... (7 actions, 6 running)
[2,519 / 2,828] GoCompilePkg external/org_golang_google_protobuf/reflect/protodesc/protodesc.a [for tool]; 0s processwrapper-sandbox ... (8 actions, 7 running)
[2,537 / 2,828] Executing genrule //pkg/sql/sem/tree:gen-createtypevariety-stringer; 0s processwrapper-sandbox ... (8 actions, 7 running)
[2,559 / 2,828] Executing genrule //pkg/util/timeutil:gen-lowercase-timezones [for tool]; 1s processwrapper-sandbox ... (8 actions, 7 running)
[2,579 / 2,828] Executing genrule //pkg/sql/catalog/lease:gen-lease-mode-stringer; 0s processwrapper-sandbox ... (8 actions, 7 running)
[2,593 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 1s processwrapper-sandbox ... (7 actions, 6 running)
[2,611 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 2s processwrapper-sandbox ... (4 actions, 3 running)
[2,616 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 3s processwrapper-sandbox ... (6 actions, 5 running)
[2,623 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 4s processwrapper-sandbox ... (8 actions running)
[2,629 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 5s processwrapper-sandbox ... (8 actions running)
[2,635 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 6s processwrapper-sandbox ... (8 actions, 7 running)
[2,646 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 7s processwrapper-sandbox ... (8 actions, 7 running)
[2,656 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 8s processwrapper-sandbox ... (7 actions running)
[2,669 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 10s processwrapper-sandbox ... (5 actions, 4 running)
[2,675 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 11s processwrapper-sandbox ... (4 actions, 3 running)
[2,680 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 12s processwrapper-sandbox ... (8 actions, 7 running)
[2,689 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 13s processwrapper-sandbox ... (5 actions running)
[2,693 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 15s processwrapper-sandbox ... (3 actions running)
[2,694 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 17s processwrapper-sandbox ... (3 actions, 2 running)
[2,702 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 18s processwrapper-sandbox ... (7 actions running)
[2,707 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 19s processwrapper-sandbox ... (3 actions running)
[2,708 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 28s processwrapper-sandbox ... (3 actions, 2 running)
[2,718 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 29s processwrapper-sandbox ... (5 actions running)
[2,721 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 31s processwrapper-sandbox ... (3 actions, 2 running)
[2,722 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 32s processwrapper-sandbox ... (2 actions running)
[2,722 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 41s processwrapper-sandbox ... (2 actions running)
[2,722 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc; 71s processwrapper-sandbox ... (2 actions running)
[2,723 / 2,828] Executing genrule //pkg/sql/parser:sql-goyacc [for tool]; 104s processwrapper-sandbox
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg2337438149 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.x -linkobj /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb.a -- /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/local_only_session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_data.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_migration.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/sessiondatapb_go_proto_/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/session_revival_token.pb.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/local_only_session_data.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/sequence_cache.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/sequence_cache_node.go /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2554/execroot/com_github_cockroachdb_cockroach/pkg/sql/sessiondatapb/session_data.go: exit status 2
Target //pkg/workload/schemachange:schemachange_test failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/workload/schemachange/BUILD.bazel:66:8 GoCompilePkg pkg/workload/schemachange/sch
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248938?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14248938?buildTab=artifacts#/) on master @ [7345fae7a2449e17f1cf93dff8ae4089d5376a8c](https://github.com/cockroachdb/cockroach/commits/7345fae7a2449e17f1cf93dff8ae4089d5376a8c):


```
[2,865 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 88s processwrapper-sandbox ... (8 actions, 7 running)
[2,879 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 89s processwrapper-sandbox ... (8 actions, 7 running)
[2,900 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 90s processwrapper-sandbox ... (8 actions, 7 running)
[2,916 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 91s processwrapper-sandbox ... (6 actions, 5 running)
[2,924 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 93s processwrapper-sandbox ... (8 actions, 7 running)
[2,929 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 94s processwrapper-sandbox ... (8 actions, 7 running)
[2,939 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 95s processwrapper-sandbox ... (8 actions, 7 running)
[2,945 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 96s processwrapper-sandbox ... (8 actions, 7 running)
[2,950 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 97s processwrapper-sandbox ... (8 actions, 7 running)
[2,956 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 98s processwrapper-sandbox ... (8 actions, 7 running)
[2,977 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 99s processwrapper-sandbox ... (8 actions running)
[2,978 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 100s processwrapper-sandbox ... (8 actions running)
[2,992 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 101s processwrapper-sandbox ... (8 actions, 7 running)
[3,017 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 102s processwrapper-sandbox ... (8 actions, 7 running)
[3,038 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 103s processwrapper-sandbox ... (8 actions, 7 running)
[3,055 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 104s processwrapper-sandbox ... (8 actions, 7 running)
[3,077 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 105s processwrapper-sandbox ... (8 actions, 7 running)
[3,090 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 106s processwrapper-sandbox ... (7 actions, 6 running)
[3,100 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 107s processwrapper-sandbox ... (8 actions, 7 running)
[3,108 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 109s processwrapper-sandbox ... (8 actions, 7 running)
[3,112 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 110s processwrapper-sandbox ... (8 actions, 7 running)
[3,122 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 111s processwrapper-sandbox ... (4 actions running)
[3,128 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 112s processwrapper-sandbox ... (8 actions, 7 running)
[3,140 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 113s processwrapper-sandbox ... (8 actions, 7 running)
[3,151 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 115s processwrapper-sandbox ... (6 actions, 5 running)
[3,156 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 116s processwrapper-sandbox ... (4 actions running)
[3,157 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 118s processwrapper-sandbox ... (4 actions, 3 running)
[3,161 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 119s processwrapper-sandbox ... (3 actions, 2 running)
[3,176 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 120s processwrapper-sandbox ... (7 actions, 6 running)
[3,190 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 121s processwrapper-sandbox ... (4 actions running)
[3,191 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 123s processwrapper-sandbox ... (3 actions running)
[3,192 / 3,583] Executing genrule //pkg/sql/parser:sql-goyacc; 130s processwrapper-sandbox ... (3 actions, 2 running)
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb/BUILD.bazel:5:11: GoCompilePkg pkg/sql/sessiondatapb/sessiondatapb.a failed: (Exit 1): builder failed: error executing command (from target //pkg/sql/sessiondatapb:sessiondatapb) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64_race -tags bazel,gss,bazel,gss,race -src ... (remaining 101 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/sql/sessiondatapb/sequence_cache_node.go:47:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:49:8: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:52:9: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
pkg/sql/sessiondatapb/sequence_cache_node.go:53:15: sc.mu.RWMutex undefined (type syncutil.RWMutex has no field or method RWMutex)
compilepkg: error running subcommand PATH=/usr/bin:/bin \
GOPATH= \
GOOS=linux \
GOROOT=bazel-out/k8-fastbuild-ST-b33d65c724e6/bin/external/io_bazel_rules_go/stdlib_ \
GOEXPERIMENT=nocoverageredesign \
CGO_ENABLED=1 \
TMPDIR=/tmp \
GOTOOLCHAIN=local \
GOROOT_FINAL=GOROOT \
GOARCH=amd64 \
external/go_sdk/pkg/tool/linux_amd64/compile -p github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -importcfg /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2917/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sessiondatapb/importcfg4206471796 -pack -race -trimpath=/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2917/execroot/com_github_cockroachdb_cockroach=>github.com/cockroachdb/cockroach/pkg/sql/sessiondatapb -o /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2917/execroot/com_github_cockr
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119910 :  failed [C-test-failure O-robot branch-release-23.1.17-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/sessiondatapb/sequence_cache_node.go
+++ b/pkg/sql/sessiondatapb/sequence_cache_node.go
@@ -44,13 +44,13 @@ func NewSequenceCacheNode() *SequenceCacheNode {
 func (sc *SequenceCacheNode) NextValue(
 	seqID catid.DescID, clientVersion uint32, fetchNextValues func() (int64, int64, int64, error),
 ) (int64, error) {
-	sc.mu.RWMutex.RLock()
+	sc.mu.RLock()
 	cacheEntry, found := sc.cache[seqID]
-	sc.mu.RWMutex.RUnlock()
+	sc.mu.RUnlock()
 
 	createSequenceCacheNodeEntry := func() {
-		sc.mu.RWMutex.Lock()
-		defer sc.mu.RWMutex.Unlock()
+		sc.mu.Lock()
+		defer sc.mu.Unlock()
 		// There is a hazard that multiple threads could add the entry, so check if it exists again with the writer lock
 		_, found = sc.cache[seqID]
 		if !found {

